frontend/src/game/card/CardBase.jsx

```diff
@@ -80,16 +87,12 @@ CardBase.propTypes = {
   card: PropTypes.object.isRequired,
 };
 
-const CardBaseImage = ({ mouseEntered, url, foil, flippedIsBack, flippedNumber, identifiers, name, setCode = "", number = "" }) => (
+const CardBaseImage = ({ imgUrl, onError, name }) => (
```

Comment:
🔥 : I might be missing something (reading on my phone), but it looks like you have removed important functionality here:
- double sided cards should flip when you mouse over them
- if an error in loading image occurs, it should try loading an alt source before falling back to text - see `getFallbackSrc`

I think I know why you were making these changes because we don't want the flipping behaviour for that corner preview?

Reply:
The cards do still flip when you hover over them, it's just handled by `CardBase` rather than `CardBaseImage`. The mouse hover functionality was implemented in both places so I went with keeping it in `CardBase` (which also let me use state for marking the error loading). I may totally be wrong about this though so feel free to correct me!

I did remove the fallback source for images though. The initial removal was because as far as my (somewhat limited) testing showed when I was first looking, it would only set the image to the fallback image temporarily since it was just setting the src on the element not the component and then whenever the component was re-rendered it would cause the whole process to start again. I'll push an update through that reimplements this though.

Reply:
Thanks! 
Sorry I didn't get an update about this comment, just reading this after revisiting the pr.

Oh good idea handling the flip at a higher level. Can then likely use it for flipping the text card a well (hmm i wonder how 2 sided cards are handled in text mode atm) 


Please @ mention me when you'd like another review / have a question @Noojuno